templates/inspec/iam_policy/properties/iam_policy_audit_configs_audit_log_configs.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: false
```

Comment:
Could these files be autogenerated?

Reply:
Could be, but it requires loading a yaml representation of the IAM policy object and running it through the compiler and including product, etc from WITHIN a compiler run. 

If you have suggestions on how to avoid the tricky yaml loading I'm open to it

Reply:
My biggest worry was that we now have two different versions of the same template (one compiled, one not).

I'm not sure I follow the YAML stuff.